### ISSUE TITLE
[FEATURE][GHG]Module: Material - Domestic Solid Waste k

### ISSUE THREAD

**Issue by guilbep** [EPFL-ENAC/unhcr-geneva-tech-hub-app#552]
<b>comment</b>: “Mixed / unknown composition”: Would it make sense to describe / explain what is meant by it? E.g. applied waste composition? Otherwise, this might not be clear to users<br/><b>actions</b>: EssentialTech will provide a text to IT4R explaining this category. Sub text to explain can be added<br/><hr/><b>estimated_hours</b>: 2<br/><b>unhcr_comments</b>: null<br/><b>ctobin111</b>: "EssentialTech to provide text"<br/><b>me</b>: "IT4R to add text"
- [ ] #578

**Comment by guilbep** [comment]
@ctobin111 waiting on the text :)

**Comment by ctobin111** [comment]
@guilbep please add the information icon to the left of mixed composition and the text should read:
"This option assumes a typical composition for non-biowaste for the country's region. Emission factors for this category can be found in the emissions factor database." 

Please provide a link to the bottom emissions factor table in the emissions factors database:

![image](https://github.com/EPFL-ENAC/unhcr-geneva-tech-hub-app/assets/111510926/424d8065-d865-4fa3-b06a-48f95e4accde)

**Comment by guilbep** [comment]
The link part is a bit more complicated: https://github.com/EPFL-ENAC/unhcr-geneva-tech-hub-app/issues/578
I've created an another issue to handle that specific problem.

Otherwise the present issue #552 is done [dev 5ec4d51] fix(domestic-solid-waste): add tooltipinfo for non-biowaste/mixed #552
